src/main/res/values-fr/strings.xml

```diff
@@ -5,4 +5,8 @@
 	<string name="locRequestOkButton">Activer</string>
 	<string name="locRequestDenyButton">Non merci</string>
 	<string name="locRequestIconDescription">Icône de mon emplacement</string>
+
+	<string name="loadingIconDescription">Chargement…</string>
+	<string name="upgradingTitle">Mettre à jour votre application</string>
+	<string name="upgradingMessage">Vous passez à la dernière version. Veuillez patienter pendant la migration de vos données.</string>
```

Comment:
@abbyad could you please check whether these french translations are OK?

Reply:
Pretty close! Minor changes suggested here:

```suggestion
	<string name="loadingIconDescription">Chargement en cours…</string>
	<string name="upgradingTitle">Mise à jour de votre application</string>
	<string name="upgradingMessage">Vous passez à la nouvelle version. Veuillez patienter pendant la preparation de vos données.</string>
```